Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3523,6 +3523,7 @@ void WebPageProxy::processNextQueuedMouseEvent()
 
 #if ENABLE(CONTEXT_MENUS)
     if (m_waitingForContextMenuToShow) {
+        WEBPAGEPROXY_RELEASE_LOG(MouseHandling, "processNextQueuedMouseEvent: Waiting for context menu to show.");
```

Comment:
Does the event type or the mouse event queue size inform anything?

Reply:
I'm not sure event types would be useful to log here. An empty event queue would cause mouse events to not be sent, but for a different reason than what I'm trying to log here.